src/Components/SearchButton.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const SearchButton = () => {
+  return <button className="btn btn-primary">Search</button>;
```

Comment:
Do you think we could make this display text configurable via props?

Reply:
do you mean like this?
` <SearchButton search={"Search"} />
const SearchButton = (props) => {
  return <button className="btn btn-primary">{props.search}</button>;
};`

Reply:
Yeap!